pkg/net/libp2p/libp2p.go

```diff
@@ -476,7 +476,21 @@ func (p *provider) bootstrap(
 		return err
 	}
 
-	bootstrapConfig := bootstrapConfigWithPeers(peerInfos)
+	ownID := p.identity.id
+	filteredPeerInfos := make([]peer.AddrInfo, 0)
+
+	for _, peerInfo := range peerInfos {
+		if peerInfo.ID == ownID {
+			logger.Infof(
+				"Found own address [%v] on bootstrap peer list. Ignoring it.",
+				peerInfo.ID,
+			)
```

Comment:
Does it make sense to log that information at all? IMO, it's enough to just `continue` in that case.

Reply:
I wasn't sure whether to add the log or not.
I added it, because the operator may be surprised that we silently remove one item from their configuration file.
If we leave the log, they will know what happened.

Reply:
After thinking about it more, I changed my mind. 
Let's not overwhelm the operator with unnecessary logs.
Removed the log and added a comment in 27d52e5125bd04b52697d25e464c9c5e112191cd.